jest.config.ts

```diff
@@ -17,13 +17,14 @@ const config: Config = {
     "<rootDir>/tests/initI18next.setup.js"
   ],
   transformIgnorePatterns: [
+    // eslint-disable-next-line max-len
     "node_modules/(?!(jest-)?@react-native|react-native|react-clone-referenced-element|@react-native-community|expo(nent)?|@expo(nent)?/.*|react-navigation|@react-navigation/.*|@unimodules/.*|unimodules|sentry-expo|native-base||(?!react-native-redash))|jest-runner"
```

Comment:
This is one where I think it would be better to change the code to confirm to the rule instead of ignoring the rule. The rule is there to make the code more readable, and you can do so here by breaking this long regex into multiple lines.